linux_os/guide/system/software/integrity/package_prelink_removed/ansible/shared.yml

```diff
@@ -0,0 +1,15 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = disable
+# complexity = medium
+# disruption = low
+
+- name: Restore Prelinked Binaries
+  ansible.builtin.shell:
+    cmd: prelink -ua
```

Comment:
I think that this should be limited only to the situation when prelink is installed.

Reply:
I have applied this suggestion to the bash remediation as well.